src/SB/Game/zEntHangable.cpp

```diff
@@ -38,8 +78,8 @@ void zEntHangable_Init(zEntHangable* ent, xEntAsset* asset)
 
     if (ent->linkCount)
     {
-        // TODO: Not quite correct
-        ent->link = (xLinkAsset*)(ent->asset + 1);
+        xEnt* next = (xEnt*)&((xEntMotion*)(ent->asset))[1]; // Not in DWARF, but anything to make this less of an eyesore.
+        ent->link = (xLinkAsset*)(&next->link); // Maybe...???
```

Comment:
This is definitely right - go ahead and remove these speculative comments, please.